### PR TITLE
Content update - print-overview (user story 1861941)

### PR DESCRIPTION
**Pull request by v-trisshores**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Comment by v-trisshores** [comment]
Hi @adegeo, both C# and VB code projects are ready for your review. Each project contains three snippets, numbered in the order in which they appear in the article.

File: dotnet-desktop-guide/net/wpf/documents/snippets/printing-overview/csharp/MainWindow.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+            // Get all print queues on the local computer.
+            PrintQueueCollection printQueueCollection = localPrintServer.GetPrintQueues();
+
+            // Return a collection of print queues, which individually hold the features or states
+            // of a printer as well as common properties for all print queues.
+            return printQueueCollection;
+        }
+        // </SampleCode2>
+
+        // <SampleCode1>
```
Comment (adegeo) [inline dotnet-desktop-guide/net/wpf/documents/snippets/printing-overview/csharp/MainWindow.xaml.cs:112]: Just FYI. In the future you can be more specific with the snippet ID. I generally use the method name:
`<GetPrintTicket>` which makes it clearer when browsing the source of the article.

File: dotnet-desktop-guide/net/wpf/documents/snippets/printing-overview/vb/MainWindow.xaml.vb [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+        Public Sub New()
+
+            InitializeComponent()
+
+        End Sub
+
+        Private Sub Grid_Loaded(sender As Object, e As RoutedEventArgs)
+
+            For Each printQueue As PrintQueue In GetPrintQueues()
+                '' Add printers that have at least one relevant print options.
```
Comment (adegeo) [inline dotnet-desktop-guide/net/wpf/documents/snippets/printing-overview/vb/MainWindow.xaml.vb:23]: VB code comments are a single ' not a double '' . I would change these to make it more VB-like

**Review by adegeo — COMMENTED** [review]
Code looks great. Thanks! Few things to note.

File: dotnet-desktop-guide/net/wpf/documents/snippets/printing-overview/vb/MainWindow.xaml.vb (hunk 2 above)
Comment (v-trisshores) [reply dotnet-desktop-guide/net/wpf/documents/snippets/printing-overview/vb/MainWindow.xaml.vb]: Done.

File: dotnet-desktop-guide/net/wpf/documents/snippets/printing-overview/csharp/MainWindow.xaml.cs (hunk 1 above)
Comment (v-trisshores) [reply dotnet-desktop-guide/net/wpf/documents/snippets/printing-overview/csharp/MainWindow.xaml.cs]: Noted, thanks.

**Comment by v-trisshores** [comment]
Hi @adegeo, the article is ready for your review.

Notes:

- For article navigation, I deliberately used article section links (#section-title) instead of XREFs on lines 64 and 72. Otherwise it's all XREFs. :)

**Pull request merged**